src/cmd_all/src/playground.rs

```diff
@@ -66,9 +66,30 @@ fn get_services(profile: &str) -> (Vec<RisingWaveService>, bool) {
         ],
         "playground-3cn" => vec![
             RisingWaveService::Meta(osstrs([])),
-            RisingWaveService::Compute(osstrs(["--host", "127.0.0.1:5687"])),
-            RisingWaveService::Compute(osstrs(["--host", "127.0.0.1:5688"])),
-            RisingWaveService::Compute(osstrs(["--host", "127.0.0.1:5689"])),
+            RisingWaveService::Compute(osstrs([
+                "--host",
+                "127.0.0.1:5687",
+                "--state-store",
+                "hummock+memory-shared",
+                "--parallelism",
+                "4",
```

Comment:
https://github.com/risingwavelabs/risingwave/issues/7293 🤔

Reply:
Should we let parallelism be the default value (total cpus available)? However, for playground-3cn it will end up with 3x total cpu available on my laptop which is too slow to run the e2e test.

Reply:
Yes, that's the problem. I think removing the `panic` in check and specifying 4 here is okay.